### PR TITLE
DTSPO-75 - add more workspaces to handle other envs and run tf fmt

### PR DESCRIPTION
**Pull request by msl8r**
### Change description ###
Adding in more environments, not sure if the mapping is 100% correct, should everything that isn't sandbox or prod go into hmcts-nonprod?

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by msl8r** [comment]
Looks like ldata is pointing to hmcts-prod - https://dev.azure.com/hmcts/CNP/_build/results?buildId=108446&view=logs&j=aa76f560-d179-5d43-b55b-3be4b1719245&t=490b43b3-a305-5552-db8f-f3db84f9bbae&l=534

and perf-test is pointing to hmcts-nonprod https://dev.azure.com/hmcts/CNP/_build/results?buildId=108446&view=logs&j=51e3a991-ec51-5e21-a220-a4a4681379e2&t=0f66e120-b983-56db-87b0-bf9a012ce5ac&l=2460

**Comment by msl8r** [comment]
Updated the mapping so ldata points to hmcts-prod and perf-test to hmcts-nonprod. Now the workspace isn't changing in the pipeline.

https://dev.azure.com/hmcts/CNP/_build/results?buildId=108456&view=results

File: main.tf [hunk 1]
```diff
@@ -23,6 +23,14 @@ locals {
       subscription_id = "1c4f0704-a29e-403d-b719-b90c34ef14c9"
       name            = "hmcts-nonprod"
     }
+    qa = {
```
Comment (timja) [inline main.tf:26]: hmm you've added qa but aren't using it? 

we don't use this anyway?
See the CFT definition where perftest uses nonprod
https://github.com/hmcts/cnp-aks-rbac-pipelines/blob/master/variablegroups/PERFTEST-AKS-Common.json#L31

can you remove the qa and ethosldata definitions, the mappings look correct

Comment (msl8r) [reply main.tf:26]: ah I missed removing that after realising it wasn't used :)

**Review by timja — APPROVED** [review]

**Pull request merged**